src/stores/AccountController.ts

```diff
@@ -487,9 +471,9 @@ export default class AccountController {
     }
 
     const [activeSubscription, transactions, activePayment] = await Promise.all([
-      this.subscriptionService.getActiveSubscription({ sandbox, customerId, config }),
-      this.subscriptionService.getAllTransactions({ sandbox, customerId }),
-      this.subscriptionService.getActivePayment({ sandbox, customerId }),
+      this.subscriptionService.getActiveSubscription({ sandbox: isSandbox, customerId, config }),
```

Comment:
Also for later: do we allow services to be "stateful"? Perhaps we can refactor out all `sandbox` properties and set the environment once via the `service#initialize()` method.

Reply:
Well, technically they are stateless as we use the store here from the outside. 
Could it make sense to make Stores pluggable in the future?

Reply:
Yes, conceptually, they should not be using stores/controllers. But there is a grey area where the Account service is configured with the environment that can also be considered state.

It would clean up some service calls, but it will also introduce a relation between different services (or every service should be initialized with the same config).

Let's note this for later.